src/servers/esl.js

```diff
@@ -17,11 +17,15 @@ class EslServer extends SocketServer {
   }
 
   bridgeCall(callId, address) {
-    return this._getSocket(callId).api('conference ' + callId + '++flags{moderator} dial sofia/gateway/partner/' + address)
+    this._getSocket(callId).event_json('CHANNEL_ANSWER');
+    this._getSocket(callId).filter('variable_conference', callId);
+    return this._getSocket(callId).api('conference ' + callId + '++flags{moderator} dial {conference=' + callId + '}sofia/gateway/partner/' + address)
   }
 
   terminateCall(callId) {
-    return this._getSocket(callId).api('conference ' + callId + ' hup all')
+    this._getSocket(callId).event_json('CHANNEL_HANGUP');
+    this._getSocket(callId).filter('variable_conference_name', callId);
+    return this._getSocket(callId).execute('conference ' + callId + ' hup all')
   }
```

Comment:
yes, freeswitch prefixes variable_ itself. 
all works. I will need to test cancelling the call before bridge though.